src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -37,11 +42,77 @@ def _raise_error_if_not_drawing_classifier_input_sframe(
     if len(dataset) == 0:
         raise _ToolkitError("Input Dataset is empty!")
 
-def create(input_dataset, feature="drawing", target="label", 
-            pretrained_model_url=None, classes=None, batch_size=256, 
-            num_epochs=100, max_iterations=0, verbose=True, **kwargs):
+def create(input_dataset, target, feature=None, 
+            pretrained_model_url=None, batch_size=256, 
+            num_epochs=100, max_iterations=0, verbose=True):
     """
     Create a :class:`DrawingClassifier` model.
+
+    Parameters
+    ----------
+    dataset : SFrame
+        Input data. The columns named by the ``feature`` and ``target``
+        parameters will be extracted for training the drawing classifier.
+
+    target : string
+        Name of the column containing the target variable. The values in this
+        column must be of string or integer type. String target variables are
+        automatically mapped to integers in the order in which they are 
```

Comment:
This seems strange to me. I thought the other toolkits all sort the unique values to obtain the list of classes and therefore the mapping

Reply:
Thanks for catching my typo. There is no `classes` parameter anymore so this doesn't make sense. We do sort the unique values behind the scenes to obtain a mapping.